src/RA_Dlg_Memory.cpp

```diff
@@ -998,11 +977,11 @@ INT_PTR Dlg_Memory::MemoryProc(HWND hDlg, UINT nMsg, WPARAM wParam, LPARAM lPara
 
         case WM_GETMINMAXINFO:
         {
-            LPMINMAXINFO lpmmi = (LPMINMAXINFO)lParam;
+            LPMINMAXINFO lpmmi      = (LPMINMAXINFO)lParam;
             lpmmi->ptMaxTrackSize.x = pDlgMemoryMin.x;
-            lpmmi->ptMinTrackSize = pDlgMemoryMin;
+            lpmmi->ptMinTrackSize   = pDlgMemoryMin;
         }
-        return TRUE;
+            return TRUE;
```

Comment:
this looks weird, recommend moving the return statement inside the brackets.

Reply:
For all of these, I'll do that.